src/assets/css/style.css

```diff
@@ -90,19 +83,8 @@ body {
   display: flex;
 }
 
-#graph-canvas-container {
```

Comment:
Same here. Reason for removal.

Reply:
Please check my fork to understand why:
https://github.com/dmx974/ComfyUI_frontend/blob/main/src/views/graph/GraphCanvas.vue